dataland-frontend/src/components/forms/CreateEuTaxonomyForFinancials.vue

```diff
@@ -540,6 +518,34 @@ export default defineComponent({
         .catch((e) => console.log(e));
     },
 
+    /**
+     * Parses received eligibility KPIs data
+     * @param receivedFormInputsModelData Received data
+     * @returns ObjectType
+     */
+    extractEligibilityKpis(receivedFormInputsModelData: ObjectType){
+      const fieldNames = Object.values(euTaxonomyKPIsModel.kpisFieldNameToFinancialServiceType);
+      const kpiKeys = Object.keys(euTaxonomyKPIsModel.kpisFieldNameToFinancialServiceType);
```

Comment:
these two variables have the same value and never change. We don't need two then.